apps/molgenis-components/src/components/forms/FormGroup.vue

```diff
@@ -3,7 +3,9 @@
     <label v-if="label !== null && label !== undefined" :for="id">
       <b> {{ label }}</b>
     </label>
-    <span v-if="required" class="float-right">(required)</span>
+    <span v-if="required === 'true' || required === true" class="float-right">
```

Comment:
I think you should also cover the case of "TRUE"

Reply:
Might be worth it to create a computed or a function for this, in order to not get too much logic in the html